### ISSUE TITLE
[Bug + Reproduction Issue] Duplicate high-frequency subband index in projection_utils.py L126, and unable to reproduce paper metrics

### ISSUE THREAD

**Issue by alicepolice** [Kidleyh/SMW-GS#3]
---

First of all, thank you for this great work! The idea of enhancing feature richness through different sampling strategies is both insightful and theoretically compelling.

While attempting to reproduce your results, I ran into two issues and would appreciate your response or a fix.

---

### Issue 1: Duplicate subband index in `projection_utils.py` line 126

The current code reads:

```python
wave_map = torch.cat(
    [Yl, Yh[0][:, :, 0, :, :], Yh[0][:, :, 0, :, :], Yh[0][:, :, 2, :, :]], dim=1
)
```

After careful inspection, **the 0th high-frequency subband `Yh[0][:, :, 0, :, :]` appears to be used twice in the original source, while the 1st subband is never used**. This means the constructed `wave_map` contains redundant information and is missing one of the intended subband features, which directly affects model performance.

Could you confirm whether the correct indices should be `0, 1, 2` (i.e., covering all three directional subbands without repetition)?

---

### Issue 2: Unable to reproduce paper metrics using hyperparameters from the supplementary material

I followed the hyperparameters provided in the supplementary material and ran multiple experiments, but was unable to reach the metrics reported in the paper — the results even fell below the GS-W baseline.

I do believe the method is valid in principle; diverse sampling strategies should logically enrich feature representations. However, given the code inconsistency above, it's hard to tell whether the gap is caused by the bug, incomplete hyperparameter descriptions, or a mismatch between the released code and the version used for the paper experiments.

**It would be very helpful if you could:**
1. Confirm and fix the subband index bug in `projection_utils.py`;
2. Verify that the released code matches the code used to produce the paper's results;
3. If possible, provide complete configs or checkpoints that reproduce the reported metrics.

Thank you, and looking forward to your reply!

---

首先，感谢你们出色的工作！论文中通过不同采样策略增强特征信息丰富度的思路非常有意思，理论上也令人信服。

在尝试复现时，我发现了以下两个问题，希望能得到回应或修复。

---

### 问题一：`projection_utils.py` 第 126 行——高频子带索引错误

当前代码如下：

```python
wave_map = torch.cat(
    [Yl, Yh[0][:, :, 0, :, :], Yh[0][:, :, 0, :, :], Yh[0][:, :, 2, :, :]], dim=1
)
```

经仔细检查，**第 0 个高频子带 `Yh[0][:, :, 0, :, :]` 被重复使用了两次，而第 1 个高频子带从未被使用**。这意味着构造出的 `wave_map` 包含了冗余信息，而丢失了应有的子带特征，会对模型性能产生直接影响。

请问正确的索引是否应该为 `0, 1, 2`（即不重复地覆盖三个方向子带）？

---

### 问题二：参考附加材料超参数，无法复现论文指标

我参照论文附加材料中提供的超参数进行了多次实验，结果均未能达到论文中声称的指标，甚至低于 GS-W 的基线水平。

我相信该工作的有效性，理论上不同的采样策略确实能够显著提升特征的信息丰富度。但当前代码状态下，很难判断差距来自上述 Bug、超参数描述不完整，还是代码与论文版本不一致。

**希望你们能够：**
1. 确认并修复 `projection_utils.py` 中的子带索引问题；
2. 检查发布代码与论文实验所用代码是否一致；
3. 如有可能，提供能稳定复现论文指标的完整配置或 checkpoint。

感谢！期待你们的回复。

**Comment by Kidleyh** [comment]
Thank you for your attention. Regarding the code issue, it seems there was an error in the version I provided, and I think your changes are correct. As for the metrics issue, I'm not entirely sure if the changes to the larger scenarios caused the metric errors in the original three scenarios. I will try to investigate the problem, but due to academic pressure, I may not be able to provide an answer quickly, for which I sincerely apologize.

**Comment by Kidleyh** [comment]
Hello, we recently fixed some bugs and submitted some ckpts.
